### PR TITLE
Lights

### PR DESCRIPTION
**Pull request by t-ekla**
Added lights class and live tested on hardware

### CODE REVIEW

File: lib/light.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      def __init__(self, redlight = 18, bluelight = 11, greenlight = 16):
+            self.redlight = redlight
+            self.bluelight = bluelight
+            self.greenlight = greenlight
+            GPIO.setmode(GPIO.BCM)
+            GPIO.setwarnings(False)
+            GPIO.setup(redlight, GPIO.OUT)
+            GPIO.setup(bluelight, GPIO.OUT)
+            GPIO.setup(greenlight, GPIO.OUT)
+
```
Comment (olliebrownlow) [inline lib/light.py:14]: Do we need extra blank line here?

File: lib/light.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+      def red(self):
+            GPIO.output(self.redlight, True)
+            GPIO.output(self.bluelight, False)
+            GPIO.output(self.greenlight, False)
+            print("Soil too dry")
+
```
Comment (olliebrownlow) [inline lib/light.py:21]: Extra blank line here too

File: lib/light.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            GPIO.output(self.bluelight, True)
+            GPIO.output(self.redlight, False)
+            GPIO.output(self.greenlight, False)
+
+      def green(self):
+            print("Soil just right")
+            GPIO.output(self.greenlight, True)
+            GPIO.output(self.redlight, False)
+            GPIO.output(self.bluelight, False)
+
```
Comment (olliebrownlow) [inline lib/light.py:34]: In the colour methods you call the print function at the top of two (blue and green) but at the bottom of the third (red) - any reason?

File: lib/light.py (hunk 1 above)
Comment (breycarr) [reply lib/light.py]: The conventions for the linter we're using recommend 2 blank lines between functions

Comment (breycarr) [reply lib/light.py:14]: 2 blank lines between functions seems to be the convention for Python, according to the [style guide](https://www.python.org/dev/peps/pep-0008/#blank-lines)

File: lib/light.py (hunk 2 above)
Comment (breycarr) [reply lib/light.py:21]: We've been inconsistent with this, in future commits we'll align with the linter and the style guide which recommend 2 blank lines after methods

File: lib/light.py (hunk 3 above)
Comment (breycarr) [reply lib/light.py:34]: We moved it down in red in order to get some visibility on why a light wasn't working, which turned out to be a hardware issue. Having this appear consistently does make more sense, so we should change that in a future commit.

**Review by laurencetaylor — COMMENTED** [review]
Code makes sense - nothing to add beyond what Ollie mentioned

**Pull request merged**